client/src/app/core/repositories/users/user-repository.service.ts

```diff
@@ -217,6 +217,7 @@ export class UserRepositoryService extends BaseRepository<ViewUser, User, UserTi
      * @param users The users to reset the passwords from
      */
     public async bulkResetPasswordsToDefault(users: ViewUser[]): Promise<void> {
+        this.preventInDemo();
```

Comment:
I added this one since it invalidates existing sessions -> everyone will be logged out.